docs/application/dotnet/guides/nui/nui-components/Button.md

```diff
@@ -160,11 +160,11 @@ When you change the Button state to focus or disable, the Button instance receiv
 
 ```cs
 Button button = new Button();
-button.StateChangedEvent += OnStateChange;
+button.ControlStateChangedEvent += OnStateChange;

```

Comment:
Just for checking. Would you please confirm "**button**.ControlStateChangedEvent" here?
I'm confused as "View.ControlStateChangedEvent" is said to replace StateChangedEvent  in Button.

Reply:
> Just for checking. Would you please confirm "**button**.ControlStateChangedEvent" here?
> I'm confused as "View.ControlStateChangedEvent" is said to replace StateChangedEvent in Button.

The class of "View" is the parent class of "Button" and "ControlStateChangedEvent"  is public, So we can use button.ControlStateChangedEvent here